grype/db/curator.go

```diff
@@ -143,7 +143,7 @@ func (c *Curator) Update() (bool, error) {
 	defer downloadProgress.SetCompleted()
 	defer importProgress.SetCompleted()
 
-	updateAvailable, updateEntry, err := c.IsUpdateAvailable()
+	updateAvailable, _, updateEntry, err := c.IsUpdateAvailable()
```

Comment:
I think we might be able to use the new `metadata` object below this line

`downloading new vulnerability DB` <-- we can add `log.Debug` level logs with the more verbose information included in the check command output

Reply:
Since `info` logs were already present, I added `metadata` info to one of them to stay consistent with the log level instead of using `log.Debug`... hope that's alright. If you feel strongly about making it debug though, just let me know and I'll fix. Thanks for reviewing!